regular-expressions/introduction/basic-syntax-i/regex-wildcards-i.md

```diff
@@ -0,0 +1,87 @@
+---
+author: emmab
+
+tags:
+  - introduction
+
+type: normal
+
+category: must-know
+
+links:
+- '[The Dot Wildcard](https://www.regular-expressions.info/dot.html){website}'
+- '[Regex metacharacters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes){website}'
+
+---
+# Wildcards I
+
+---
+## Content
+
+Wildcards can be used to add flexibility when you don't want to specify characters. 
+
+> In regex, the `.` wildcard can be used to stand in for nearly *any* character[1] which is why it's the most powerful wildcard.
+
+`/abc.*/`
+
+Using the above pattern, we'd be able to match the following strings because they all start with `'abc'` and then have zero or more other characters.
+
+'**abc**' ✅
+'**abc1**' ✅
+'**abc!f@g£jf$h%^**' ✅
+
+> If we want to match only *word characters*, like letters, numbers and underscores, then we can use `\w`.
+
+`/\w/`
```

Comment:
same here, use h3 (`###`) for headline as it increased font size and applies bold
```suggestion
### `/\w/`
```